core/server/data/migrations/versions/2.18/1-restore-settings-from-backup.js

```diff
@@ -0,0 +1,85 @@
+const _ = require('lodash');
+const Promise = require('bluebird');
+const common = require('../../../../lib/common');
+const config = require('../../../../config');
+const fs = require('fs-extra');
+const path = require('path');
+
+module.exports.config = {
+    transaction: true
+};
+
+const backupFileRegex = /ghost.([\d]{4}-[\d]{2}-[\d]{2}).json$/;
+
+module.exports.up = (options) => {
+    const contentPath = config.get('paths').contentPath;
+    const dataPath = path.join(contentPath, 'data');
+
+    const localOptions = _.merge({
+        context: {internal: true}
+    }, options);
+
+    return fs.readdir(dataPath).then(function (files) {
+        const backups = files.filter(function (filename) {
+            return backupFileRegex.test(filename);
+        }).sort(function (a, b) {
+            const dateA = new Date(a.match(backupFileRegex)[1]);
+            const dateB = new Date(b.match(backupFileRegex)[1]);
+
+            return dateB - dateA;
+        });
+
+        // CASE: One we _just_ made for the migration
+        const mostRecentBackup = backups[0];
+        // CASE: The previous one from last migration
+        const secondMostRecentBackup = backups[1];
```

Comment:
I think this can actually be removed because we have the new format

Reply:
yeah. if we know what the old & new format is, we can focus on old 👍